install.sh

```diff
@@ -120,6 +120,42 @@ nvm_detect_profile() {
   fi
 }
 
+#
+# Check whether the user has any globally-installed npm modules in their system
+# Node, and warn them if so.
+#
+nvm_check_global_modules() {
```

Comment:
Please ensure that any `nvm_*` functions defined are also added to the `unset -f` lists in this file, and/or in `nvm.sh`, whichever is appropriate.